airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/debezium/CdcSourceTest.java

```diff
@@ -823,4 +834,7 @@ protected AirbyteCatalog expectedCatalogForDiscover() {
     return expectedCatalog;
   }
 
+  protected void waitForCdcRecords(String schemaName, String tableName, int recordCount)
+      throws Exception {}
```

Comment:
If you want to be really pedantic about stacked changes shouldn't you be adding some kind of //TODO or //XXX comment here?

Reply:
that's not a todo. It's just a hook for other tests to insert some wait in here.